INSTALL.md

```diff
@@ -31,6 +31,8 @@ CONFIG_HAVE_BPF_JIT=y
 CONFIG_BPF_EVENTS=y
 ```
 
+For Linux kernel version 4.7 or newer, the flag `CONFIG_HAVE_BPF_JIT` has been replaced by `CONFIG_HAVE_EBPF_JIT` (for x86, arm64 and s390) or by `CONFIG_HAVE_CBPF_JIT` (for arm, mips, powerpc and sparc.)
```

Comment:
why is it relevant to users who install bcc?

Reply:
FYI, `INSTALL.md` still says to look for CONFIG_HAVE_BPF_JIT. My kernel's configuration doesn't have that, but it does have `CONFIG_HAVE_EBPF_JIT`.

Reply:
Since you are looking at this, could you help submit a pull request to replace `CONFIG_HAVE_BPF_JIT` with `CONFIG_HAVE_EBPF_JIT`? Thanks!